{
  "PublicDateAtUSN": "0001-01-01T00:00:00Z",
  "CRD": "0001-01-01T00:00:00Z",
  "Candidate": "CVE-2020-16092",
  "PublicDate": "2020-07-29T21:26:00Z",
  "References": [
    "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-16092",
    "https://access.redhat.com/security/cve/CVE-2020-16092"
  ],
  "Description": "QEMU: reachable assertion failure in net_tx_pkt_add_raw_fragment() in hw/net/net_tx_pkt.c",
  "UbuntuDescription": "",
  "Notes": null,
  "Bugs": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=1860283"
  ],
  "Priority": "medium",
  "DiscoveredBy": "Ziming Zhang",
  "AssignedTo": "",
  "Patches": {
    "qemu": {
      "bionic": {
        "Status": "needs-triage",
        "Note": ""
      },
      "devel": {
        "Status": "needs-triage",
        "Note": ""
      },
      "focal": {
        "Status": "needs-triage",
        "Note": ""
      },
      "precise/esm": {
        "Status": "DNE",
        "Note": ""
      },
      "trusty": {
        "Status": "ignored",
        "Note": "out of standard support"
      },
      "trusty/esm": {
        "Status": "needs-triage",
        "Note": ""
      },
      "upstream": {
        "Status": "needs-triage",
        "Note": ""
      },
      "xenial": {
        "Status": "needs-triage",
        "Note": ""
      }
    },
    "qemu-kvm": {
      "bionic": {
        "Status": "DNE",
        "Note": ""
      },
      "devel": {
        "Status": "DNE",
        "Note": ""
      },
      "focal": {
        "Status": "DNE",
        "Note": ""
      },
      "precise/esm": {
        "Status": "needs-triage",
        "Note": ""
      },
      "trusty": {
        "Status": "DNE",
        "Note": ""
      },
      "trusty/esm": {
        "Status": "DNE",
        "Note": ""
      },
      "upstream": {
        "Status": "needs-triage",
        "Note": ""
      },
      "xenial": {
        "Status": "DNE",
        "Note": ""
      }
    }
  },
  "UpstreamLinks": {}
}